tools/cldr-code/src/main/java/org/unicode/cldr/util/CoreCoverageInfo.java

```diff
@@ -35,14 +35,7 @@
     public enum CoreItems {
         // Drop the exemplars, since
         // main_exemplar, auxiliary_exemplar, // numbers_exemplar, punctuation_exemplar, index_exemplar(Level.MODERN)
-        orientation,
-        plurals,
-        default_content, likely_subtags,
-        country_data,
-        casing,
-        collation,
-        romanization(Level.MODERATE),
-        ordinals(Level.MODERN),
+        orientation, plurals, default_content, likely_subtags, country_data, casing, collation, romanization(Level.MODERATE), ordinals(Level.MODERN),
```

Comment:
This is too intrusive. Assume the author had good reason to put things on separate lines.

Reply:
Indeed. Sadly I cannot get the Eclipse formatter to always handle this sort of thing properly, even when the settings should do so (i.e. the exact same code is formatted correctly in the formatting test window).
Worse, the IntelliJ plugin seems not to work reliably either.

I'm beginning to think the only option for "trouble free, just format everything" maintenance is the Google formatter, but that's only an option if everyone is happy with what it does (please have a look at those PRs if you have a moment).